### PR TITLE
Move lock history data to separate model [#23]

### LINKED ISSUE #23: move lock history data to separate model
The lock history should not be saved in the audits table. Instead, a separate model should be created.

### PR DESCRIPTION
**Pull request by leewaa**
closes #23

### CODE REVIEW

File: app/models/foreman_dlm/dlmlock_event.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module ForemanDlm
+  class DlmlockEvent < ApplicationRecord
+    include Authorizable
```
Comment (timogoebel) [inline app/models/foreman_dlm/dlmlock_event.rb]: if we include `Authorizable`, we also have to define permissions for this.

File: app/views/foreman_dlm/dlmlocks/_details.html.erb [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
+      <th><%= _("User") %></th>
+      <th><%= _("Type") %></th>
       <th><%= _("Timestamp") %></th>
-      <th><%= _("Enabled") %></th>
     </tr>
   </thead>
   <tbody>
-    <% @dlmlock.audits.includes(:user).reorder(created_at: :desc).each do |audit| %>
-      <% revision = audit.revision %>
+    <% @dlmlock.dlmlock_events.includes(:user).reorder(created_at: :desc).each do |event| %>
```
Comment (timogoebel) [inline app/views/foreman_dlm/dlmlocks/_details.html.erb]: We need to check if the user is authorized to see events.

File: app/models/foreman_dlm/dlmlock_event.rb (hunk 1 above)
Comment (leewaa) [reply app/models/foreman_dlm/dlmlock_event.rb]: Done

**Comment by timogoebel** [comment]
@leewaa: Please rebase.

**Comment by timogoebel** [comment]
Thanks, @leewaa.

**Pull request merged**